### PR TITLE
Devex-1017: build-client-integration test intermittent failures

### PR DESCRIPTION
**Pull request by rosiehsmith**
[Associated trello card
](https://trello.com/c/dgva54sP/1017-investigate-circle-build-inconsistency-integration-tests-random-failure)

This PR (1/x) specifically addresses the integration test `web-client/integration-tests/noticeOfChangeOfAddressQCJourney.test.js` like in this failed[ CircleCI workflow ](https://app.circleci.com/pipelines/github/ustaxcourt/ef-cms/3273/workflows/a9e97ff5-7321-47a4-9f16-2f13d9f4a115/jobs/33095) as well as [noticeOfTrialSessionWithPaperService.test.js](https://app.circleci.com/pipelines/github/ustaxcourt/ef-cms/3279/workflows/f915f318-811e-4176-9a50-91d0a0a31ac8/jobs/33207) and [noticeOfTrialSessionWithElectronicService.test.js](https://app.circleci.com/pipelines/github/ustaxcourt/ef-cms/3290/workflows/2d82b0b0-7cd5-4d90-9654-3cfa1f7e1b56/jobs/33363).

**Suspected cause of intermittency**: now that we are using `isAsync` on our async endpoints, the tests are not waiting until they're done processing and immediately return a 204

**Changes**: We're now waiting for specific items other than the loading spinner, like the progress bar to complete and disappear when updating the practitioner address which addresses the async `updateUserContactInformationLambda`. For noticeOfTrialSessions tests, we're waiting for the correct page and alertSuccess.

Also noticed an import was wrong in `web-client/src/presenter/sequences/loadMorePendingItemsSequence.js` so fixed that since it failed while building the docker image created to test these locally.

### CODE REVIEW

File: docs/testing.md [hunk 1]
```diff
@@ -275,4 +275,12 @@ All of the expected output images are found in the `./shared/test-pdf-expected-i
 docker build -t "ef-cms-us-east-1:pdf-compare" -f Dockerfile-pdf-testing .
 docker run -it -v `pwd`/shared/test-output:/home/app/efcms/shared/test-output ef-cms-us-east-1:pdf-compare sh -c "cd efcms && ./update-pdf-images.sh"
 cp -r shared/test-output/*.1.png shared/test-pdf-expected-images/
+```
+
+## Client Integration Testing
```
Comment (mrinsin) [inline docs/testing.md:280]: This is awesome!

**Review by mrinsin — APPROVED** [review]
alt text

**Review by TomElliottFlexion — APPROVED** [review]

**Comment by TomElliottFlexion** [comment]
https://media.giphy.com/media/QZgzJIU5ODel9D1Hob/giphy.gif

**Comment by TomElliottFlexion** [comment]
alt text

**Review by halprin — APPROVED** [review]
I'm late but this is great!

**Pull request merged**